src/Parser/Hunter/BeastMastery/Modules/Spells/BestialWrath/BestialWrath.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS/index';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatNumber } from 'common/format';
+import getDamageBonus from "Parser/Hunter/Shared/Core/getDamageBonus";
+import StatisticBox from "Main/StatisticBox";
+import SpellIcon from "common/SpellIcon";
+import ITEMS from "common/ITEMS/HUNTER";
+
+const T204P_MODIFIER = 0.15;
+const T20_2P_MODIFIER_PR_STACK = 0.015;
+const BESTIAL_FURY_MODIFIER = 0.15;
+const BESTIAL_WRATH_BASE_MODIFIER = 0.25;
+let BESTIAL_WRATH_TOTAL_MODIFIER = 0;
+
+const AFFECTED_ABILITIES = [
+  SPELLS.KILL_COMMAND.id,
+  SPELLS.COBRA_SHOT.id,
+  SPELLS.MULTISHOT.id,
+];
+
+const debug = false;
+
+class BestialWrath extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  bonusDmg = 0;
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    //resets the strength of bestial wrath every time it's cast and adjust for Bestial Fury talent.
+    if (spellId === SPELLS.BESTIAL_WRATH.id) {
+      if (this.combatants.selected.hasTalent(SPELLS.BESTIAL_FURY_TALENT) || this.combatants.selected.hasFinger(ITEMS.SOUL_OF_THE_HUNTMASTER.id)) {
+        BESTIAL_WRATH_TOTAL_MODIFIER = BESTIAL_FURY_MODIFIER + BESTIAL_WRATH_BASE_MODIFIER;
+      } else {
+        BESTIAL_WRATH_TOTAL_MODIFIER = BESTIAL_WRATH_BASE_MODIFIER;
+      }
+
+    }
+    if (spellId !== SPELLS.KILL_COMMAND.id || spellId !== SPELLS.COBRA_SHOT.id || spellId !== SPELLS.MULTISHOT.id) {
+      return;
+    }
+
+    if (this.combatants.selected.hasBuff(SPELLS.HUNTER_BM_T20_2P_BONUS.id) && this.combatants.selecteded.hasBuff(SPELLS.BESTIAL_WRATH.id)) {
```

Comment:
How about checking for the 2PT20 bonus in the on_initialized call? It doesn't change during the fight.

Reply:
Why would I put it there? Would I simply say 
``if (this.combatants.selected.hasBuff(SPELLS.HUNTER_BM_T20_2P_BONUS.id) {
this.T20_2P_MODIFIER_PR_STACK = 0.015 
}`` and then otherwise have it set as 0?